README.md

```diff
@@ -119,7 +119,7 @@ If you intend to actively contribute to Hasjob code, some functionality is sourc
 
 Finish configuration with:
 
-    $ python manage.py db create
+    $ flask db create
```

Comment:
Again, this does not exist. Not sure where it was coming from before.

Reply:
The Docker instructions are obsolete and should be removed, along with related files.